tests/detectors/test_detectors_continuation.py

```diff
@@ -29,7 +29,7 @@ def test_continuation_load():
 def test_continuation_1_matching_trigger():
     d = _plugins.load_plugin("detectors.continuation.Continuation")
 
-    a = garak.attempt.Attempt(prompt="test prompts")
+    a = garak.attempt.Attempt(prompt=garak.attempt.Message(text="test prompts"))
```

Comment:
Many of the edits in this file look ripe for a single `CONSTANT`.

Reply:
at suite-level, even. added module level one here